src/components/home/Social.jsx

```diff
@@ -1,5 +1,28 @@
-const Social = ({ name, link }) => {
-  return <div>Social</div>;
+import Link from "next/link";
+
+import { BsEnvelope, BsFacebook } from "react-icons/bs";
+import { LINKS } from "@/data/links";
+
+const Social = ({ link = LINKS }) => {
+  return (
+    <div className="flex flex-row">
+      <Link
```

Comment:
You can map through the LINKS
try something like this 
<img width="464" alt="image" src="https://github.com/acm-ucr/ewb-website/assets/73259022/03c7f29b-7002-4962-b878-cf73889b94bb">